DemoAppSwiftUI/DemoAppSwiftUIApp.swift

```diff
@@ -13,6 +13,7 @@ struct DemoAppSwiftUIApp: App {
     @Injected(\.chatClient) public var chatClient: ChatClient
     
     @ObservedObject var appState = AppState.shared
+    @ObservedObject var notificationsHandler = NotificationsHandler.shared
```

Comment:
I'm always a little confused by this but shouldn't this be a `@StateObject`? 

The way I understand it is that the View who's creating/holding the created object should have it as a `@StateObject` and the inheriting ones as a `@ObservedObject`.

Reply:
It's a shared instance, so it's not owned by the view. Only in this case it can be @ObservedObject, otherwise it should be @StateObject.